product_docs/docs/postgres_distributed_for_kubernetes/1/known_issues.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Known issues and limitations'
+---
+
+These known issues and limitations are in the current release of EDB Postgres Distributed for Kubernetes. 
+
+## Upgrades 
+
+This version of EDB Postgres Distributed for Kubernetes **does not support** migrating from existing Postgres databases, nor upgrading 
```

Comment:
Agree with the migration part.
But upgrades when we upgrade to a newer patch release should work.
It's major version upgrades that we don't support.